content/blog/2022/11/2022-11-14-hacktoberfest-recap.adoc

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: "A Wonderful Hacktoberfest in Jenkins"
+tags:
+- hacktoberfest
+- event
+- community
+- newcomer
+- outreach-programs
+author: kmartens27
+description: >
+  Hacktoberfest 2022 was a great success for the Jenkins project.
+  This post highlights Jenkins contributions in artwork, translation, documentation, security, and new features.
+links:
+  discourse: https://community.jenkins.io/t/hacktoberfest-2022/3805
+opengraph:
+  image: /images/hacktoberfest/hacktoberfest_2022_social.png
+---
+
+As November continues, we want to share some highlights from Hacktoberfest 2022!
+This year's edition was incredibly busy with lots of interesting ideas, great work, new Jenkins contributors, and returning contributors. 
+We want to thank all of the contributors and mentors for all of their hard work, as none of this would be possible without them.
+
+link:https://hacktoberfest.com/[Hacktoberfest] is an event to highlight and celebrate open-source projects like Jenkins.
+The main idea of the event is contributing to and improving open-source software.
+People are invited to submit pull requests to contribute to plugins, provide documentation, create translations, and review code among other things, to engage with open-source software. 
+Many products and services are based on open-source software, so giving back in this way continues the cycle of development.
+
+For link:https://www.jenkins.io/events/hacktoberfest/faq/[Jenkins], Hacktoberfest is an opportunity to collaborate with and expand the community, while learning and sharing with all experience levels.
+These contributions are what keeps Jenkins moving forward, so having an opportunity to celebrate both the project and the people is amazing.
+
+== Hacktoberfest results
+
+During October, the Jenkins project received over 613 pull requests from 117 different contributors.
+All of these helped to improve Jenkins, create new resources, and review the needs of Jenkins accordingly.
```

Comment:
I'm not sure of the meaning of the phrase "review the needs of Jenkins".  Optional alternative for your consideration
```suggestion
All of these helped to improve Jenkins, create new resources, and meet the needs of Jenkins users.
```